kornia/augmentation/augmentations.py

```diff
@@ -160,26 +182,34 @@ def __repr__(self) -> str:
             hue={self.hue}, return_transform={self.return_transform})"
         return self.__class__.__name__ + repr
 
-    def forward(self, input: UnionType) -> UnionType:  # type: ignore
+    @staticmethod
+    def get_params(batch_size: int, brightness: FloatUnionType = 0., contrast: FloatUnionType = 0.,
+                   saturation: FloatUnionType = 0., hue: FloatUnionType = 0.) -> Dict[str, torch.Tensor]:
+        return pg._random_color_jitter_gen(batch_size, brightness, contrast, saturation, hue)
+
+    def forward(self, input: UnionType, params: Optional[Dict[str, torch.Tensor]] = None) -> UnionType:  # type: ignore
+        if params is None:
+            if isinstance(input, tuple):
+                batch_size = input[0].shape[0] if len(input[0].shape) == 4 else 1
+            else:
+                batch_size = input.shape[0] if len(input.shape) == 4 else 1
```

Comment:
suggest to add a function to infer the batch size since is used in every transform